src/components/patch-assignment-form/utils/edit-patch-assignment.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Patch,
+  ResponsibleEntity,
+  replacePatchResponsibleEntities,
+} from "@mtfh/common/lib/api/patch/v1";
+
+export function editPatchAssignment(
+  patch: Patch,
+  newOfficerName: string,
+  newOfficerEmail: string,
+  handleSubmission: Function,
+) {
+  const officer = patch?.responsibleEntities[0];
+
+  const newResEnt: ResponsibleEntity = {
+    ...officer,
+    name: newOfficerName,
+    contactDetails: {
+      emailAddress: newOfficerEmail,
+    },
+  };
+
+  const currentVersionNumber = patch.versionNumber || 0;
+  // console.log(`${patch.name} ${patch.versionNumber} || ${newResEnt.name} -> ${newResEnt.contactDetails?.emailAddress}`);
```

Comment:
Remove

Reply:
Removed in 08ab2e4d31f804e0ca32f11ab9dd1a3ac9387e54